test/integration/build_admission_test.go

```diff
@@ -166,8 +167,9 @@ func setupBuildStrategyTest(t *testing.T) (clusterAdminClient, projectAdminClien
 	imageStreamMapping.Tag = "latest"
 	imageStreamMapping.Image.Name = "image-id"
 	imageStreamMapping.Image.DockerImageReference = "test/builderimage:latest"
-	err = clusterAdminClient.ImageStreamMappings(testutil.Namespace()).Create(imageStreamMapping)
-	if err != nil {
+	if err = unversioned.RetryOnConflict(unversioned.DefaultRetry, func() error {
+		return clusterAdminClient.ImageStreamMappings(testutil.Namespace()).Create(imageStreamMapping)
```

Comment:
it's super weird to make a client care about retrying a conflict on a create... what have we done that makes this necessary?